### DISCUSSION TITLE
App Icon Proposition

### DISCUSSION THREAD

**Discussion by bobogringo** [deckerst/aves#607] [Ideas]
Hi, first of all I discovered your app three days ago and I am aaaaamazed. It's great; it's everything I wanted from a gallery app and I thought it did not exist until now. 

A big thank you for your amazing work. 

The only thing I don't like that much is the icon app so I spent my sunday morning trying to make something else on the theme of birds. I almost replicated your icon actually, only more in a google vibe.

If it is to your liking, I could send you a .ai or .svg file, or any format usefull to you.

![aves](https://user-images.githubusercontent.com/79135941/233840400-eba07065-aa72-4039-b65b-4560d4948f21.png)

**Comment by bobogringo** [comment]
![aves_2](https://user-images.githubusercontent.com/79135941/233853339-b7ed08fe-ef5c-4a25-9740-26c40eedb762.png)

Another version, this time really ripping off the google suite design.

I don't even know if this would be legally ok to use.

**Comment by deckerst** [comment]
I'm glad you enjoy the app.

Thanks for your interest in the design of the app, but this repo does not accept icon/asset submission, just like it does not accept code.

By the way, the app is *not* trying to fit into Google esthetics. Otherwise I would stick to material design and google primary colors.
